pages/micromaster/profiles/tkim949.md

```diff
@@ -0,0 +1,48 @@
+## Taekyoung Kim
+##### Seattle, WA || PST || Windows 10 - DeskTop, macOS Catalina(Version 10.15.7) - LapTop
+##### [Linkedin](https://www.linkedin.com/in/tkim949), [Github](https://github.com/tkim949)
+
+### School:
+##### Oregon State University  
+###### Expected Graduation in Dec. 2020
+<!--- (![alt text](https://github.com/tkim949/tkim949.github.io/blob/tkprofile/pages/micromaster/images/OSU.png "Oregon State University")) -->
```

Comment:
as this line is commented out image file should be removed.

Reply:
I removed the line with comment

Reply:
Also removed the school logo image